src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.jsx

```diff
@@ -126,6 +126,12 @@ const GroupDetails = createReactClass({
           Sentry.withScope(scope => {
             Sentry.captureException(new Error('Project not found'));
           });
+        } else {
+          const location_with_project = {...browserHistory.getCurrentLocation()};
+          if (!location_with_project.query.project) {
+            Object.assign(location_with_project.query, {project: project.id});
```

Comment:
Do you need `Object.assign` to update a single property? Would `locationWithProject.query.project = project.id` work?